Topic 3/Topic3 - Ex 1,2,3/src/test/java/com/globant/Topic3/service/ShoppingCartServiceTest.java

```diff
@@ -0,0 +1,55 @@
+package com.globant.Topic3.service;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+import com.globant.Topic3.model.Product;
+
+public class ShoppingCartServiceTest {
+	
+	private final ShoppingCartServiceImp cartImp = new ShoppingCartServiceImp();
+	
+	@Test

```

Comment:
This Unit Test suite includes only possitive tests, what about the negative test? I mean, the cases of add and remove a null object are not covered.